src/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component, PureComponent } from "react";
+import ReactDOM from "react-dom";
+
+class HelloWorld extends Component {
+    render() {
+        return (
+            <h1>Hello World! [Component]</h1>
+        );
+    }
+}
+
+const HelloWorld2 = () => {
+    return <h1>Hello World! [func style]</h1>
+}
+
+class HelloWorld3 extends Component {
+    shouldComponentUpdate(nextProps, nextState) {
+        //do nothing by now
+    }
+    render() {
+        return (
+            <h1>Hello World! [Pure Component]</h1>
+        );
+    }
+}
+
+ReactDOM.render(<div><HelloWorld /> <HelloWorld2 /> <HelloWorld3 /></div>, document.getElementById("index"));
+ReactDOM.render(React.createElement("h1", null, "Hello World! [createElement]"), document.getElementById("index2"));
```

Comment:
why do you need to roots?

Reply:
well, it was because I used `createElement` outside of any class, and I do not find how to mix `jsx` and function call.
Btw, I rewrite it into a class and and now it's possible to use in 1 root. But I think it's not well example to use `createElement` if we use it inside `Component` class (I found this in react docs).